### PR TITLE
feat(icons): Add Heart icon

### PR DESCRIPTION
**Pull request by rivka-ungar**
https://monday.monday.com/boards/3532714909/views/80492480/pulses/6754531709

### CODE REVIEW

File: packages/core/src/components/Icon/Icons/components/Heart.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+      fill="#676879" fillRule="evenodd" clipRule="evenodd" />
```
Comment (talkor) [inline packages/core/src/components/Icon/Icons/components/Heart.tsx:10]: The fill should be currentColor no?

Comment (talkor) [reply packages/core/src/components/Icon/Icons/components/Heart.tsx:10]: @rivka-ungar you should re-generate the icons to update this one as well

Comment (rivka-ungar) [reply packages/core/src/components/Icon/Icons/components/Heart.tsx]: oops

File: packages/style/plop/icon/index.js [hunk 2]
```diff
@@ -23,13 +25,37 @@ module.exports = plop => {
...
     ],
-    actions: [
-      {
-        type: "modify",
-        path: "src/Icons/iconsMetaData.js",
-        pattern: /(\/\/ plop_marker:icon_metadata)/g,
-        templateFile: "plop/icon/icon-metadata.txt"
-      }
-    ]
+    actions: () => {
```
Comment (talkor) [inline packages/style/plop/icon/index.js:28]: Amazing

**Review by talkor — APPROVED** [review]
🎉

**Pull request merged**